### DISCUSSION TITLE
Shows, Episode remover from Home Screen.

### DISCUSSION THREAD

**Discussion by fndrsm** [SpotX-Official/SpotX#17] [Ideas]
Hey dev team, i wonder if u can remove the Shows or Episode from the home screen of the spotify, really annoying that spotify not make the hide feature yet. I never listening to episodes or podcast or anything except music, and they are really annoying keep showing up on my home screen. 

Like this ( i dont know what based recomendation for this show, the cover really nsfw for me)
![image](https://user-images.githubusercontent.com/48046968/144027115-c7a39021-d998-417c-b98b-2e9c27b73fc8.png)

I read about article for removing this thing, but i dont understand what he did, because they are using mac, not windows, i dont know which the code i have to remove. 
This is the article [https://remysharp.com/2021/08/17/removing-shows-from-spotify](https://remysharp.com/2021/08/17/removing-shows-from-spotify)

I read his command, `sed 's/,show,/,/'` i think it is to remove the 'word' and change to '/'.
I did this and the spotify not showing anything hahaha.

But if u considering this, i would say many thanks for what u did. Cheers

**Comment by amd64fox** [comment]
For a long time there was an idea to remove podcasts, thanks for the idea😀, I will answer you a little later.

**Reply by amd64fox** [reply]
@fndrsm 
Added disabling podcasts (optional),

**Reply by fndrsm** [reply]
@amd64fox the besttttt, manyy thanksss !! IT WORKKSS!!
![image](https://user-images.githubusercontent.com/48046968/144197980-a35f21af-3798-42a8-a198-fdd142cfe59b.png)
